source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -312,7 +316,10 @@ MobileHttpConnectionManagerFilterConfigFactory::createFilterFactoryFromProtoType
         EnvoyMobileHttpConnectionManager& mobile_config,
     Server::Configuration::FactoryContext& context) {
   return HttpConnectionManagerFilterConfigFactory::createFilterFactoryFromProtoAndHopByHop(
-      mobile_config.config(), context, false);
+      mobile_config.config(), context,
+      mobile_config.config().has_clear_hop_by_hop_response_headers()
+          ? mobile_config.config().clear_hop_by_hop_response_headers().value()
+          : false);
```

Comment:
Prefer `PROTOBUF_GET_WRAPPED_OR_DEFAULT`, ditto in all other call sites.

Reply:
Thanks! Already done.